From b087eadb1a932eb4bb4468cf84d6b85b05d06070 Mon Sep 17 00:00:00 2001
From: Patrick Ohly <patrick.ohly@intel.com>
Date: Tue, 12 May 2015 01:39:00 -0700
Subject: [PATCH] ima: fix configuration of policy loading

IMA_POLICY_REPLACEABLE and IMA_LOAD_POLICY depend on IMA_POLICY_LOADER,
which did not exist. As a result it was impossible to enable these
two options.

Upstream-Status: Submitted [http://permalink.gmane.org/gmane.comp.handhelds.tizen.devel/6279]

Signed-off-by: Patrick Ohly <patrick.ohly@intel.com>
---
 security/integrity/ima/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/security/integrity/ima/Kconfig b/security/integrity/ima/Kconfig
index b31e3f7..2834cb2 100644
--- a/security/integrity/ima/Kconfig
+++ b/security/integrity/ima/Kconfig
@@ -158,6 +158,13 @@ config IMA_APPRAISE_SIGNED_INIT
 	help
 	   This option requires user-space init to be signed.
 
+config IMA_POLICY_LOADER
+        bool "IMA policy loading"
+        default n
+        help
+          Enabling this option grants controll over a variety of
+          options related to loading policy rules from files.
+
 config IMA_POLICY_REPLACEABLE
 	bool "Allows to replace policy at runtime"
 	depends on IMA_POLICY_LOADER
-- 
1.8.4.5

